### ISSUE TITLE
Export image des réseaux

### ISSUE THREAD

**Issue by paulgirard** [medialab/toflit18#107]
export images + légende des réseaux en PNG ?

**Comment by gdaudin** [comment]
.svg comme les autres images, plutôt ? Pas de bouton : mettre le même que dans, par exemple, Time Series ?

**Comment by sim51** [comment]
Ajout du bouton export avec le menu pour le choix entre CSV et SVG (comme sur les timeries).
Ajout de la légende avec la couleur des noeuds

Voici le résultat pour la page "Locations" :
![image](https://user-images.githubusercontent.com/231657/94682636-8978ca00-0325-11eb-85c8-2240107ed31a.png)

**Comment by gdaudin** [comment]
Ce serait bien d’avoir un peu plus de légende avec ? I.e. ce à quoi correspondent le size des nodes et la thickness des edges.

**Comment by sim51** [comment]
Voici ce que cela donne avec la légende en entière : 
![image](https://user-images.githubusercontent.com/231657/94790013-fc417e00-03d5-11eb-8a10-781b9c791264.png)

Pour information, je ne peux pas enlever de manière simple l'aspect des champs.

@gdaudin  Est-ce que cela vous convient ?

**Comment by gdaudin** [comment]
Good enough. Et une petite phrase statique pour la couleur des nœuds ?

Il faudrait mettre les paramètres du filtre dans le nom du fichier ?

**Comment by sim51** [comment]
Pour la couleur des noeuds, c'est déjà présent dans la légende (ci-dessus `color : Community louvain`, ou directement la couleur avec la typologie du noeud pour l'export de la page "Locations".

En ce qui concerne le nom du fichier, le nom du fichier est le suivant : `TOFLIT18_Product_terms_2020-10-01_11 04 19.svg`. 
C'est le même nom que celui de l'export en CSV. Il faudrait donc faire la modification sur les deux noms de fichier pour être homogène.

Ce développement bien que simple à réaliser, consomme pas mal de temps vu le nombre de paramètre à prendre en compte avec la gestion de leur présence ou non.

Je vous propose donc de voir en fin de projet s'il reste du temps après la réalisation des autres taches, qui me semblent plus prioritaires (mais je peux me tromper).

**Comment by gdaudin** [comment]
Oui, tout à fait. Esprit d’escalier : ce serait peut-être bien de rajouter en bas du fichier image le nom du fichier en texte (en fait, surtout Toflit / la date / les filtres dans l’idéal -- mais dans l’idéal ce serait le nom du fichier. Ou l’URL ?)

Mais ce n’est pas prioritaire en effet.

**Comment by gdaudin** [comment]
Pourquoi est-ce que dans l’exporte, il y a la boîte "exports export csv / svg" ? C’est pareil dans les exports de TimesSeries, d’ailleurs. 
<img width="1162" alt="image" src="https://user-images.githubusercontent.com/10850348/94993690-18d6e500-0593-11eb-92d2-66be3aa35404.png">

**Comment by gdaudin** [comment]
![Uploading image.png…]()

**Comment by gdaudin** [comment]
Ah ! Mais c’est un bug spécifique à Firefox. Cela marche avec Safari et Chrome. On peut laisser tomber, mais dans ce cas-là, il faudrait signaler l’incompatibilité quelque part.

**Comment by gdaudin** [comment]
on renonce. Dommage de ne pas avoir trouvé le temps de signaler l’incompatibilité.
